site/docs/hive.md

```diff
@@ -0,0 +1,62 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Hive
+
+## Hive read support
+Iceberg supports the reading of Iceberg tables from [Hive](https://hive.apache.org) by using a [StorageHandler](https://cwiki.apache.org/confluence/display/Hive/StorageHandlers). 
+
+### Table creation
+This section explains the various steps needed in order to overlay a Hive table "on top of" an existing Iceberg table.
+
+#### Create an Iceberg table
```

Comment:
Should we update the Hive metastore integration to automatically set up the correct `StorageHandler` so HMS tables are automatically readable? That seems reasonable for 0.10.0.

Reply:
Created a pull request: #1505

Reply:
I'll wait to see where the above goes as the results of this will probably need documenting here too - how to set up the hive-site.xml config params etc.